### DISCUSSION TITLE
quick navigation for audio files (and other media?)

### DISCUSSION THREAD

**Discussion by simonz21** [QL-Win/QuickLook#1291] [Ideas]
I use this tool lots of times every day. It makes Windows 10x more useful. However I spend a lot of my time going through audio files. On the Mac there are jump 15 seconds forward/back buttons and these are incredibly useful. Can we have the same here please, and link them to the forward/back cursor arrows too please?

**Comment by Grooves1210** [comment]
Skip forward/backwards for media files would be great!

It can't be the cursor arrow keys as these are currently used to change which file is selected for preview*
 
*depending on the view in file explorer (icons, details, list view, etc) 
eg:  up/down= previous/next  in details view
left/right = previous next in icons view
